framework/src/base/Assembly.C

```diff
@@ -296,7 +371,7 @@ Assembly::reinitFE(const Elem * elem)
 
       fesd->_phi.shallowCopy(const_cast<std::vector<std::vector<Real> > &>(fe->get_phi()));
       fesd->_grad_phi.shallowCopy(const_cast<std::vector<std::vector<RealGradient> > &>(fe->get_dphi()));
-      if (_need_second_derivative[fe_type])
```

Comment:
I'm pretty sure you guys do plenty of that without the need for a board!

What's wrong with this though?  It doesn't set it true...